pkg/sink/initialization.go

```diff
@@ -41,6 +41,10 @@ var (
 		"The namespace of the EventListener resource for this sink.")
 	portFlag = flag.String("port", "",
 		"The port for the EventListener sink to listen on.")
+	tlsCertFlag = flag.String("tls-cert", "",
+		"Filename for the TLS certificate.")
```

Comment:
Is this the filename or path to the file?

Reply:
earlier it was path to the file.
But after design change its now filename.